cli/create/templates/cartridge/failover.yml

```diff
@@ -0,0 +1,8 @@
+# Specify here the failover parameters that will
+# be used by the `cartridge failover` command
+
+mode: <mode>
```

Comment:
Make `stateful` failover with `stateboard` state provider by default. It should work:

```
cartridge start -d
cartridge replicasets setup --bootstrap-vshard
cartridge failover setup
``` 

After running these commands cluster should have no issues.